_src/static/sass/utilities/_variables.scss

```diff
@@ -25,6 +25,8 @@ $baseline-horizontal:    $baseline;
 // color: config
 $transparent:       rgba(0, 0, 0, 0);
 
+$white:             rgb(255, 255, 255);
+$black:             rgb(0, 0, 0);
 $gray:              rgb(127, 127, 127);
 $red:               rgb(178, 6, 16);
 $yellow:            rgb(237, 189, 60);
```

Comment:
See my notes above about using values already in the Sass color maps. There is one for yellow.